### DISCUSSION TITLE
Setting up new HA location/server.  Cameras are discovered but not alarm panel or sensors like at my other location

### DISCUSSION THREAD

**Discussion by Deanerh** [tsightler/ring-mqtt#807] [Support Q&A]
First, I’m not an expert so I basically just try and follow the instructions for add-ons and integrations,.  I have the MQTT broker running, but can only get the cameras to show and the video is just the last captured one recorded, not Live.  Bigger issue is the alarm system isn’t discovered at all.

**Comment by tsightler** [comment]
Discovered devices are listed in the logs during startup, most of the time when people complain about missing devices it's due to HA UI caching and nothing to do with ring-mqtt.  HA UI sometimes doesn't display devices when a lot of devices are added quickly.  Usually just force browser refresh and they will appear.  I would check the discovery logs when ring-mqtt starts up and see what it shows.

**Comment by Deanerh** [comment]
Given the fact that I’m such a newb, I’m not sure how much help I can be but I’ll give it a shot.  When you say everything shows up, when you look at your Device list, is a Keypad and Base Station showing along with the cameras?  Also, just to be sure (and this is probably a stupid question for you) do you just have the ring-mqtt add-on installed or did you also install the “official”  Ring Integration?

I managed to get everything working this morning on my end.  Turns out I wasn’t following the instructions for getting MQQT set up and working right.  I got that done and then reinstalled the ring-mqtt add-on and was able to see in the log that it was connecting correctly to MQTT but I still wasn’t getting any device discovery.  Finally looking at the log again I figured it out.  I have 4 locations on my Ring account (2 shared).  One of the shared (and the first alphabetically)  is my daughter’s apartment and the Ring unit was offline.  Once the add-on connects to the Ring account it starts looking for devices at each location but since the first one it was coming to was off-line, it was hanging right there.  Got that base station on-line and updated this morning and BAM, 66 devices at 4 locations.  Alarm panel works.

________________________________
